File: C/unit_5/Be.c
#include <stdio.h>
int main()
{
    int n, x, y, res = 0;
    printf("Enter the number : \n");
    scanf("%d", &n);
    x = n;
    while (x != 0)
    {
        y = x % 10;
        res += y * y * y;
        x /= 10;
    }
    if (res == n)
        printf("The number %d is an Armstrong Nymber \n", n);
    else
        printf("The number %d is not an Armstrong Nymber \n", n);

    return 0;
}

File: Java/unit_5/Be.java
package Java.unit_5;

import java.util.Scanner;

public class Be {
    public static void main(String[] args) {
        int n, x, y, res = 0;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Number : \n");
        n = sc.nextInt();
        x = n;
        while (x != 0) {
            y = x % 10;
            res += y * y * y;
            x /= 10;
        }
        if (res == n)
            System.out.println("The number " + n + " is an Armstrong Number : ");
        else
            System.out.println("The number " + n + " isn't an Armstrong Number : ");
        sc.close();
    }

}
